### PR TITLE
Project structure update

### PR DESCRIPTION
**Pull request by beamhafsa**
1. Removed previously committed .idea folder
2. Updated project structure
3. Added few test cases for LMS (Earn and Burn APIs)
4. Updated GetByBeacon feature files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
+target
```
Comment (MJaghoub) [inline .gitignore]: Please add global .gitignore instead of adding it in the project

File: .idea/workspace.xml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (MJaghoub) [inline .idea/workspace.xml]: Can we remove this file

File: src/test/java/me/beam/apiautomations/steps/LMSAPIStepDef.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+  private Response lmsResponse;
+  private JsonPath lmsResponseJson;
+  private String lmsBody;
+
+  private HttpRequestHelper httpHelper = new HttpRequestHelper();
+
+  private String baseURL = "http://aef8dbbd6294a11e9a2a4021e775f7f0-2112399806.eu-west-1.elb.amazonaws.com/lms/";
+  private String lmsJson = "src/test/resources/files/LMS.json";
+  private final static String authToken = "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzUxMiJ9.eyJpYXQiOjE0OTkwMDA2MjUsImV4cCI6NDY1MjY"
```
Comment (MJaghoub) [inline src/test/java/me/beam/apiautomations/steps/LMSAPIStepDef.java]: We need to work on this to generate token

File: .idea/workspace.xml (hunk 2 above)
Comment (beamhafsa) [reply .idea/workspace.xml]: Removed this file in the latest commit

**Pull request merged**